js/js_test.go

```diff
@@ -467,7 +467,7 @@ func TestCallWithNull(t *testing.T) {
 func TestReflection(t *testing.T) {
 	o := js.Global.Call("eval", "({ answer: 42 })")
 	if reflect.ValueOf(o).Interface().(*js.Object) != o {
-		t.Fail()
+		t.Fatal()
```

Comment:
AFAIK, `Fatal()` is for reporting a failure message.  Maybe `FailNow()` would be more semantically correct?